### PR TITLE
Specify motion.create functionality concerning additional_submitter 

### LINKED ISSUE #2753: Motion.create changes
See OpenSlides/openslides-client#4405

- New setting to activate/deactivate `additional_submitter` field in `motion.create`
- Don't use the caller as submitter if only an `additional_submitter` but no actual submitter id was given

### PR DESCRIPTION
**Pull request by luisa-beerboom**
Closes #2753 
Needs OpenSlides/openslides-meta#192
Needed for OpenSlides/openslides-client#4405

### CODE REVIEW

File: tests/system/action/motion/test_create.py [hunk 1]
```diff
@@ -107,9 +111,37 @@ def test_create_simple_fields(self) -> None:
...
+        assert motion.get("submitter_ids") is None
+
+    def test_create_additional_submitter_forbidden(self) -> None:
+        self.set_models(
+            {
+                "motion/1": {
+                    "title": "title_eJveLQIh",
+                    "meeting_id": 1,
+                },
+                "meeting/1": {"mediafile_ids": [8], "meeting_mediafile_ids": [80]},
```
Comment (hjanott) [inline tests/system/action/motion/test_create.py:126]: Media files and motion/1 shouldn't be necessary.

File: tests/system/action/motion/test_create.py [hunk 2]
```diff
@@ -107,9 +111,37 @@ def test_create_simple_fields(self) -> None:
             },
         )
         self.assert_status_code(response, 200)
-        self.assert_model_exists(
+        motion = self.assert_model_exists(
             "motion/2", {**motion, "attachment_meeting_mediafile_ids": [80]}
         )
+        assert motion.get("submitter_ids") is None
+
+    def test_create_additional_submitter_forbidden(self) -> None:
```
Comment (hjanott) [inline tests/system/action/motion/test_create.py:119]: Add in this test: the motion being created with submitter and additional submitter.

File: docs/actions/motion.create.md [hunk 3]
```diff
@@ -56,7 +56,8 @@ This is the logic for other fields depending on the motion type:
 
 There are some fields that need special attention:
 - `workflow_id`: If it is given, the motion's state is set to the workflow's first state. The workflow must be from the same meeting. If the field is not given, one of the three default (`meeting/motions_default_workflow_id` or `meeting/motions_default_amendment_workflow_id`) workflows is used depending on the type of the motion to create.
-- `submitter_ids`: These are **user ids** and not ids of the `submitter` model. If nothing is given (`[]`), the request user's id is used. For each id in the list a `motion_submitter` model is created. The weight must be set to the order of the given list.
+- `additional_submitter` a text field where text-based submitter information may be entered. Cannot be set unless `meeting/motions_create_enable_additional_submitter_text` is `true`.
+- `submitter_ids`: These are **user ids** and not ids of the `submitter` model. If nothing is given (`[]`) adn the field `additional_submitter` isn't filled, the request user's id is used. For each id in the list a `motion_submitter` model is created. The weight must be set to the order of the given list.
```
[the remark this answers is not in the document]
Comment (luisa-beerboom) [reply docs/actions/motion.create.md:60]: Done

File: tests/system/action/motion/test_create.py (hunk 1 above)
Comment (luisa-beerboom) [reply tests/system/action/motion/test_create.py]: Done

File: tests/system/action/motion/test_create.py (hunk 2 above)
Comment (luisa-beerboom) [reply tests/system/action/motion/test_create.py]: Added it as a separate test

**Review by hjanott — APPROVED** [review]

**Pull request merged**